### PR TITLE
Add greenkeeper ignore section to package.lock

### PR DESCRIPTION
**Pull request by gesinger**
## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/liwecukasi/edit?html,output))
- [x] Reviewed by Two Core Contributors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -140,5 +140,14 @@
     "videojs-swf": "^5.2.0",
     "watchify": "^3.6.0",
     "webpack": "^1.13.2"
+  },
+  "greenkeeper": {
+    "ignore": [
+      "browserify",
+      "qunitjs",
+      "sinon",
```
Comment (forbesjo) [inline package.json]: Should we still let GK keep updating the qunit and sinon dependencies?

Comment (gesinger) [reply package.json]: Sinon will break, not sure offhand about qunit.

Comment (gkatsev) [reply package.json]: updating qunit is probably fine.

Comment (gkatsev) [reply package.json]: Video.js couldn't because it had to support IE8 and qunit 2 dropped support for it but contrib-hls never supported IE8.

**Pull request merged**